run-in-retro-term.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# cool-retro-term support for tetris.sh
+# Author: Rojen Zaman <rojen@riseup.net>
+# License: WTFPL
+
+stdout="/dev/null"                                              # default stdout, optional verbose mode is /dev/stdout
+script_dir=$(dirname $(readlink -f $0))                         # script dir, it is require for cool-retro-term
+
+# usage
+usage() {
+    printf "%s\n" \
+        "usage : $0 [-h] [-v]" \
+        " -h   help" \
+        " -v   verbose"
+    exit 0
+}
+
+# check cool-retro-term and xdg-open commands
+check_commands() {
+    local message=()
+    command -v xdg-open &>$stdout || message+="xdg-open could not be found, please install it"
+    command -v cool-retro-term &>$stdout || message+="cool-retro-term could not be found, please install it"
+    [ ${#message} == 0 ] && return 0
+    printf "%s\n" "${message[@]}"
+    echo "Are you sure you are using GNU operating system ;-)?"
+    exit 1
+}
+
+# music loop
+run_music() {
+    while :
```

Comment:
Let's not waste lines :)
```
while : ; do
```
or
```
while true; do
```
for better readability

Reply:
I've already done it in the code I write now :)